src/utils/ledger.js

```diff
@@ -125,6 +157,61 @@ if (IPC_RENDERER) {
       }
       IPC_RENDERER.send("ledger:getSignatures", arr);
     },
+
+    /**
+     * Sign token info on ledger
+     *
+     * @param {Object} token
+     *  with uid (hex), symbol and name
+     *
+     * @memberof Ledger
+     * @inner
+     */
+    signToken(token) {
```

Comment:
Question: The expected flow here would be to:
1. sign a number of tokens
2. send them to ledger
3. verify their signatures
4. send a tx using the tokens (which is not implemented here yet, right?)

Is that right?

In which cases do we use the `resetTokenSignatures`?

Reply:
The flow to sign a tx will be:

1. Individually sign custom tokens that will be used
2. When user clicks to send transaction
3. Send custom tokens if they exist (if not, go to 5)
4. If they fail, show which failed
5. Send the `signTx` command 

The `sendToken` actually verifies the token signature and prepares the context for the `signTx`. So we don't need to call verify here. It will only be used on startup to check the signatures on storage are still valid.